containers/jupyter-datascience-notebooks/README.md

```diff
@@ -0,0 +1,32 @@
+# Jupyter Data Science Notebooks
+
+## Summary
+
+Jupyter's [Data Science Notebook container](https://hub.docker.com/r/jupyter/datascience-notebook),
+converted here into a VS Code DevContainer.
+
+| Metadata | Value |  
+|----------|-------|
+| *Contributors* | Nathan Carter |
```

Comment:
```suggestion
# Jupyter Data Science Notebooks (Community)

## Summary

*Use Jupyter Data Science Notebooks with Python, R, Julia, and more.*

| Metadata | Value |  
|----------|-------|
| *Contributors* | Nathan Carter (@nathancarter) |
```

 This section is used by VS Code and codespaces to add a heading into drop down for Open Folder in Container / Reload Folder / Add Development Container Configuration files, so links will not work. Here's a suggested tweak. Same comment on (Community) here as well.
 
 It would also be great to include your GitHub ID so we can easily "@" you into any issues that may be raised.